posts/draft.md

```diff
@@ -0,0 +1,174 @@
+# A deep dive into the MyST world
+
+> *This is a series of blog posts inviting you to join me in a little journey I have experienced in the last few weeks at the time to figure it out a nice story for **MyST**
```

Comment:
Just a meta thought - I like using the markdown approach of _linebreak after every sentence_ so that you don't run into semi-arbitrary line breaks like this

Reply:
Mmm... I need to try it... I see how linebreak after every sentence could be nice for some people and really ugly for others. I promise I will eventually make a try with another piece of text :wink:

Reply:
in case it's helpful, I've found this site to be a useful rationale for doing line breaks this way :-) https://sembr.org/

they actually recommend more complex rules for deciding when to break lines, but I think that "one line per sentence" is easy enough to follow